dev/tests/static/framework/Magento/TestFramework/Integrity/Library/Injectable.php

```diff
@@ -67,7 +61,7 @@ public function getDependencies(FileReflection $fileReflection)
      * @return ReflectionClass|null
      * @throws ReflectionException
      */
-    private function getParameterClass(ReflectionParameter $reflectionParameter): ?ReflectionClass
+    public function getParameterClass(ReflectionParameter $reflectionParameter): ?ReflectionClass
```

Comment:
I don't see external usages of this method (except the unit test). What is the reason of access level change?

If this is needed only for the test, can we update it to avoid mocking this method and test the functionality of `getDependencies` method as a whole?
```suggestion
    private function getParameterClass(ReflectionParameter $reflectionParameter): ?ReflectionClass
```

Reply:
hi @sivaschenko , yes, it was needed only for the test.
I changed the approach for this test, please check the last commit.